spec/chewy/type/import_spec.rb

```diff
@@ -43,6 +43,45 @@
       end
     end
 
+    describe 'using scopes with includes' do
+      if const_defined?('::ActiveRecord') && ::ActiveRecord::VERSION::MAJOR > 4
```

Comment:
Unfortunatly the `:active_record` tag isn't enough here, because the scope does use `.where.not` which only works in ActiveRecord >= 4, so I have to guard this even for the ActiveRecord gemfiles. But if I don't check that `ActiveRecord` is present with this `const_defined?`, the spec will fail with `uninitialized constant ActiveRecord` for the non-AR gemfiles.